src/components/interactive-builder/add-question-modal.component.tsx

```diff
@@ -580,6 +604,76 @@ const AddQuestionModal: React.FC<AddQuestionModalProps> = ({
                   />
                 </RadioButtonGroup>
               ) : null}
+
+              {questionType === 'programState' && (
```

Comment:
![CleanShot 2024-05-31 at 12  00 46@2x](https://github.com/openmrs/openmrs-esm-form-builder/assets/8509731/2e4cc9d9-6b5a-42fa-9091-56c53aabf5c4)

Can we add some margin between the fields?